plugins/documentation/docs/de_de/module-simsearch.md

```diff
@@ -15,32 +15,43 @@ Sollte eine Suche keine Ergebnisse liefern, füllt Search it das Result-Array mi
 ```php
 <?php
 $request = rex_request('search', 'string', false);
+$article = rex_article::getCurrent();
+$sim_limit = 10; // Maximales Limit ähnlicher Suchbegriffe
 
 if($request) { // Wenn ein Suchbegriff eingegeben wurde
     $server = rtrim(rex::getServer(), "/");
 
-    print '<section class="search_it-hits">';
-
-    // Init search and execute
+    // Suche wie initieren und ausführen
     $search_it = new search_it();
     $result = $search_it->search($request);
 
-    echo '<h2 class="search_it-headline">Suchergebnisse</h2>';
-    if($result['count'] == 0 && count($result['simwords']) > 0){
-        // Ähnlichkeitssuche ausgeben
-        $newsearchString = $result['simwordsnewsearch'];
-        $result_simwords = $search_it->search($newsearchString);
-        if($result_simwords['count'] > 0){
-            echo '<p>Meinten Sie <strong>'. $newsearchString .'</strong>?</p>';
-        }
-    }
-
     if($result['count']) {
```

Comment:
Ist das hier jetzt nicht total unnötig, wenn unten die Ähnlichkeitssuche ausgegeben wird? Ergibt das Beispiel so überhaupt Sinn? Ich verstehe es jetzt nicht mehr.

Reply:
Du meinst die Zeilen `if($result['count']) {` bis Ende des passenden `else`? Die müssen nicht mit rein. War früher schon drin. Ich finde es sinnvoll. So kann man anhand der anderen Codebeispiele einordnen wo der Beispielcode hin gehört. Auf Wunsch mach ich das aber raus.

Reply:
Mir ist jetzt nicht klar, ob Ähnlichkeitssuche-Treffer zusätzlich zur regulären Treffern ausgegeben wird oder nicht. Die Ziellogik sollte sein, dass die Ähnlichkeitssuche entweder zur greift, wenn die reguläre keine Treffer hat, oder diese ergänzt. Aber das geht mir jetzt aus dem Beispiel nicht hervor. Kann sein, dass es von mir schon nicht so gut gelöst war.

Man beachte auch den Version-7-Branch, in den die Änderungen dann mit eingepflegt werden müssen.

Reply:
Hier im Beispiel ist es so, dass die Ähnlichkeitssuche nur angezeigt wird, wenn keine Treffer gefunden wurden und es auch ähnliche Begriffe mit Treffern gibt. Sie soll nicht angezeigt werden, wenn es reguläre Treffer gibt.

Reply:
Dann sollte man noch den Kommentar verbessern, dass man hier eigene Logik noch verbauen muss..

Reply:
Kein Problem. Passt das so wie hier: https://github.com/FriendsOfREDAXO/search_it/pull/304

Reply:
Das macht es besser, zu verstehen, dass man selbst noch aktiv werden muss. Gut gut :) Danke

Reply:
Danke fürs drüber schauen!